packages/react/src/ActionList/ActionList.examples.stories.tsx

```diff
@@ -133,6 +134,11 @@ export function MixedSelection(): JSX.Element {
     {text: 'Due Date', icon: <CalendarIcon />},
   ]
 
+  const clearGroup = () => {
+    ;(listRef.current?.querySelector('li[aria-selected="true"]') as HTMLLIElement | undefined)?.focus()
```

Comment:
Opted to focus the last previously-selected item here after clear, but can update it to be the first/last element or whatever other element is preferred. Wondering if @primer/design-reviewers have any input here CC: @maximedegreve